JDP190901-3

pawban opened the pull request:
Kontroler został uproszczony - teraz zwraca po prostu przykładowe dane.
Zgodnie z sugestią Pawła Pluty kontroler zawiera metodę generującą przykładowe dane, z których korzystają niektóre metody endpointów.
W klasie CartDto zamieniłem konstruktory i gettery na stosowne adnotacje Lomboka.

pawban committed:
CartController - wstępna implementacja endpointów (poprawiona)

Kontroler został uproszczony - teraz zwraca po prostu przykładowe dane.
W klasie CartDto zamieniłem konstruktory i gettery na stosowne
andotacje Lomboka.

Arelias reviewed (approved):
Metody zwracaja dummy data i sa sformulowane tak, jak trzeba, aby pozniej dzialaly z DB

pawban committed:
Merge branch 'JDP190901-4' into JDP190901-3

mryndak commented on src/main/java/com/kodilla/ecommercee/CartController.java:
```diff
@@ -1,101 +1,68 @@
+
+    @PostMapping("/{id}/orders")
+    public OrderDto createOrder(@PathVariable Long id, @RequestBody OrderDto order) {
+        return new OrderDto(
```
Bardziej logiczne na ten moment było by zwrócenie tego OrderDto co przyszło

pawban replied:
Dokonałem zmian zgodnie z Twoją sugestią.

pawban committed:
CartController - poprawiona metoda createOrder()

Zgodnnie z sugestią Mateusza na tym etapie metoda ctreateOrder()
zwraca po prostu ten obiekt, któsy dostała.
